control-plane/agents/core/src/core/resource_map.rs

```diff
@@ -0,0 +1,53 @@
+use common_lib::types::v0::store::UuidString;
+use std::{
+    collections::HashMap,
+    hash::Hash,
+    sync::{Arc, Mutex},
```

Comment:
Please use the locking primitives from parking_lot...

Reply:
Done.